document_page_approval/models/document_page.py

```diff
@@ -57,17 +56,20 @@ class DocumentPage(models.Model):
         string="Approver groups",
         help="Groups that can approve changes to this document",
         compute="_compute_approver_group_ids",
+        recursive=True,
     )
 
     has_changes_pending_approval = fields.Boolean(
         compute="_compute_has_changes_pending_approval",
-        string="Has changes pending approval",
```

Comment:
Isn't this changing the casing of the string to `Has Changes Pending Approval`?

Reply:
Without this change, when executing pre-commit the following log is shown:

`The attribute string is redundant. String parameter equal to name of variable`

Reply:
Yeah, but that's not my question? If you remove it, what is the label? If it's with capital letters, then the pylint check is not correct. If it's the same, then my bad and everything is correct.

Reply:
Ok sorry, removing the string tag odoo auto-defines it with "_Has Changes Pending Approval_" (wrong). The pylint validation seems to be wrong.

Reply:
Please put the issue in pylint-odoo project.